front/app/containers/Admin/reporting/utils/getReportWidth.ts

```diff
@@ -0,0 +1,13 @@
+interface Params {
+  smallerThanPhone: boolean;
+  smallerThanTablet: boolean;
+}
+
+export const getReportWidth = ({
+  smallerThanPhone,
+  smallerThanTablet,
+}: Params) => {
+  if (smallerThanPhone) return 'mobile';
```

Comment:
We use the word `phone` in `useBreakpoint('phone')`. Why did you use `mobile` here? Could we make it consistent?

Reply:
Or maybe it would be even more convenient to have a hook like `useBreakpointName` that returns `'phone' | 'tablet' | 'desktop'`.

Reply:
Good point. I'll change it to `phone`.

If it turns out that we need to do this very often I'll indeed consider creating a `useBreakpointName` hook... but for now it's just in two places (and only in one of those places I had to call the `useBreakpoint` hook twice just for this purpose)